### DISCUSSION TITLE
Error: Invalid CSS. How to fix it?

### DISCUSSION THREAD

**Discussion by omg777** [alshedivat/al-folio#1791] [Q&A] [answered]
```
omg777.github.io/_sass/_base.scss:499: Error: Invalid CSS after "        max-height": expected "}", was ": 200px" (SassC::SyntaxError)
        on line 499:9 of _sass/_base.scss
        from line 6:1 of main.scss
>>         max-height: 200px

```
I deployed project with wrong correction in _sass/_base.scss file. After that, even if I correct it properly and depoly it, the wrong version is still loaded and an error occurs. The '_sass/_base.scss' file is also automatically modified to the wrong version. How can I solve this problem?

**Comment by george-gca** [comment, marked as answer]
In your browser do a refresh with clean cache. Press [Shift + F5 on Chrome](https://support.google.com/chrome/answer/157179#zippy=%2Cwebpage-shortcuts) or [Ctrl + F5 on Firefox](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly) to reload the page ignoring cached content, clean your browser history, or simply try it in a private session. See if that works.
